2024-09-15 21:45:44,771 INFO    StreamThr :31652 [internal.py:wandb_internal():85] W&B internal server running at pid: 31652, started at: 2024-09-15 21:45:44.770202
2024-09-15 21:45:44,779 DEBUG   HandlerThread:31652 [handler.py:handle_request():158] handle_request: status
2024-09-15 21:45:44,781 INFO    WriterThread:31652 [datastore.py:open_for_write():87] open: E:\DeepPurposePlusPlus\wandb\run-20240915_214544-prcpm8iw\run-prcpm8iw.wandb
2024-09-15 21:45:44,781 DEBUG   SenderThread:31652 [sender.py:send():379] send: header
2024-09-15 21:45:44,955 DEBUG   SenderThread:31652 [sender.py:send():379] send: run
2024-09-15 21:45:45,421 INFO    SenderThread:31652 [dir_watcher.py:__init__():211] watching files in: E:\DeepPurposePlusPlus\wandb\run-20240915_214544-prcpm8iw\files
2024-09-15 21:45:45,422 INFO    SenderThread:31652 [sender.py:_start_run_threads():1188] run started: prcpm8iw with start time 1726429544.772401
2024-09-15 21:45:45,426 DEBUG   HandlerThread:31652 [handler.py:handle_request():158] handle_request: check_version
2024-09-15 21:45:45,426 DEBUG   SenderThread:31652 [sender.py:send_request():406] send_request: check_version
2024-09-15 21:45:45,749 DEBUG   HandlerThread:31652 [handler.py:handle_request():158] handle_request: run_start
2024-09-15 21:45:46,243 DEBUG   HandlerThread:31652 [system_info.py:__init__():26] System info init
2024-09-15 21:45:46,243 DEBUG   HandlerThread:31652 [system_info.py:__init__():41] System info init done
2024-09-15 21:45:46,243 INFO    HandlerThread:31652 [system_monitor.py:start():194] Starting system monitor
2024-09-15 21:45:46,244 INFO    SystemMonitor:31652 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-09-15 21:45:46,244 INFO    HandlerThread:31652 [system_monitor.py:probe():214] Collecting system info
2024-09-15 21:45:46,248 INFO    SystemMonitor:31652 [interfaces.py:start():188] Started cpu monitoring
2024-09-15 21:45:46,248 INFO    SystemMonitor:31652 [interfaces.py:start():188] Started disk monitoring
2024-09-15 21:45:46,249 INFO    SystemMonitor:31652 [interfaces.py:start():188] Started gpu monitoring
2024-09-15 21:45:46,255 INFO    SystemMonitor:31652 [interfaces.py:start():188] Started memory monitoring
2024-09-15 21:45:46,262 INFO    SystemMonitor:31652 [interfaces.py:start():188] Started network monitoring
2024-09-15 21:45:46,266 DEBUG   HandlerThread:31652 [system_info.py:probe():152] Probing system
2024-09-15 21:45:46,267 DEBUG   HandlerThread:31652 [system_info.py:_probe_git():137] Probing git
2024-09-15 21:45:46,343 DEBUG   HandlerThread:31652 [system_info.py:_probe_git():145] Probing git done
2024-09-15 21:45:46,343 DEBUG   HandlerThread:31652 [system_info.py:probe():200] Probing system done
2024-09-15 21:45:46,343 DEBUG   HandlerThread:31652 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.22631-SP0', 'python': '3.9.18', 'heartbeatAt': '2024-09-15T19:45:46.267683', 'startedAt': '2024-09-15T19:45:44.763054', 'docker': None, 'cuda': None, 'args': ('--target_encoding', 'DGL_GCN', '--seed', '7', '--lr', '0.00001', '--wandb_proj', 'DeepPurposePP', '--epochs', '60'), 'state': 'running', 'program': 'E:\\DeepPurposePlusPlus\\train\\subcellular.py', 'codePathLocal': 'train\\subcellular.py', 'codePath': 'train\\subcellular.py', 'git': {'remote': 'https://github.com/jiaqingxie/ProB.git', 'commit': 'c395098df42c737d4df2cf289cb7095e18dd08b9'}, 'email': 'jiaxie@student.ethz.ch', 'root': 'E:/DeepPurposePlusPlus', 'host': 'DESKTOP-E5HJ25M', 'username': '嘉庆', 'executable': 'E:\\anaconda3\\envs\\graph\\python.exe', 'cpu_count': 16, 'cpu_count_logical': 24, 'cpu_freq': {'current': 2100.0, 'min': 0.0, 'max': 2100.0}, 'cpu_freq_per_core': [{'current': 2100.0, 'min': 0.0, 'max': 2100.0}], 'disk': {'/': {'total': 214.84276962280273, 'used': 73.30562591552734}}, 'gpu': 'NVIDIA GeForce RTX 4070 Laptop GPU', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4070 Laptop GPU', 'memory_total': 8585740288}], 'memory': {'total': 15.745964050292969}}
2024-09-15 21:45:46,343 INFO    HandlerThread:31652 [system_monitor.py:probe():224] Finished collecting system info
2024-09-15 21:45:46,344 INFO    HandlerThread:31652 [system_monitor.py:probe():227] Publishing system info
2024-09-15 21:45:46,344 DEBUG   HandlerThread:31652 [system_info.py:_save_conda():209] Saving list of conda packages installed into the current environment
2024-09-15 21:45:46,346 ERROR   HandlerThread:31652 [system_info.py:_save_conda():223] Error saving conda packages: [WinError 2] 系统找不到指定的文件。
Traceback (most recent call last):
  File "E:\anaconda3\envs\graph\lib\site-packages\wandb\sdk\internal\system\system_info.py", line 216, in _save_conda
    subprocess.call(
  File "E:\anaconda3\envs\graph\lib\subprocess.py", line 349, in call
    with Popen(*popenargs, **kwargs) as p:
  File "E:\anaconda3\envs\graph\lib\subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "E:\anaconda3\envs\graph\lib\subprocess.py", line 1436, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] 系统找不到指定的文件。
2024-09-15 21:45:46,372 DEBUG   HandlerThread:31652 [system_info.py:_save_conda():224] Saving conda packages done
2024-09-15 21:45:46,372 INFO    HandlerThread:31652 [system_monitor.py:probe():229] Finished publishing system info
2024-09-15 21:45:46,385 DEBUG   SenderThread:31652 [sender.py:send():379] send: files
2024-09-15 21:45:46,385 INFO    SenderThread:31652 [sender.py:_save_file():1454] saving file wandb-metadata.json with policy now
2024-09-15 21:45:46,435 INFO    Thread-16 :31652 [dir_watcher.py:_on_file_created():271] file/dir created: E:\DeepPurposePlusPlus\wandb\run-20240915_214544-prcpm8iw\files\conda-environment.yaml
2024-09-15 21:45:46,435 INFO    Thread-16 :31652 [dir_watcher.py:_on_file_created():271] file/dir created: E:\DeepPurposePlusPlus\wandb\run-20240915_214544-prcpm8iw\files\wandb-metadata.json
2024-09-15 21:45:47,212 INFO    wandb-upload_0:31652 [upload_job.py:push():130] Uploaded file C:\Users\嘉庆\AppData\Local\Temp\tmpau46bik8wandb\ajh5hkcy-wandb-metadata.json
2024-09-15 21:45:48,277 ERROR   gpu       :31652 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-15 21:45:48,277 ERROR   gpu       :31652 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-15 21:45:49,986 DEBUG   HandlerThread:31652 [handler.py:handle_request():158] handle_request: python_packages
2024-09-15 21:45:49,986 DEBUG   SenderThread:31652 [sender.py:send_request():406] send_request: python_packages
2024-09-15 21:45:49,987 DEBUG   HandlerThread:31652 [handler.py:handle_request():158] handle_request: status_report
2024-09-15 21:45:49,989 DEBUG   HandlerThread:31652 [handler.py:handle_request():158] handle_request: stop_status
2024-09-15 21:45:49,989 DEBUG   SenderThread:31652 [sender.py:send_request():406] send_request: stop_status
2024-09-15 21:45:49,994 DEBUG   HandlerThread:31652 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-15 21:45:50,166 DEBUG   SenderThread:31652 [sender.py:send():379] send: telemetry
2024-09-15 21:45:50,166 DEBUG   SenderThread:31652 [sender.py:send():379] send: config
2024-09-15 21:45:50,487 INFO    Thread-16 :31652 [dir_watcher.py:_on_file_created():271] file/dir created: E:\DeepPurposePlusPlus\wandb\run-20240915_214544-prcpm8iw\files\output.log
2024-09-15 21:45:50,487 INFO    Thread-16 :31652 [dir_watcher.py:_on_file_created():271] file/dir created: E:\DeepPurposePlusPlus\wandb\run-20240915_214544-prcpm8iw\files\requirements.txt
2024-09-15 21:45:50,659 ERROR   gpu       :31652 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-15 21:45:50,659 ERROR   gpu       :31652 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-15 21:45:52,503 INFO    Thread-16 :31652 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\DeepPurposePlusPlus\wandb\run-20240915_214544-prcpm8iw\files\output.log
2024-09-15 21:45:53,049 ERROR   gpu       :31652 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-15 21:45:53,050 ERROR   gpu       :31652 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-15 21:45:55,548 DEBUG   HandlerThread:31652 [handler.py:handle_request():158] handle_request: status_report
2024-09-15 21:45:59,452 ERROR   gpu       :31652 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-15 21:45:59,452 ERROR   gpu       :31652 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-15 21:46:00,579 DEBUG   HandlerThread:31652 [handler.py:handle_request():158] handle_request: status_report
2024-09-15 21:46:01,837 ERROR   gpu       :31652 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-15 21:46:01,837 ERROR   gpu       :31652 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-15 21:46:05,626 DEBUG   HandlerThread:31652 [handler.py:handle_request():158] handle_request: status_report
2024-09-15 21:46:05,919 INFO    MainThread:31652 [internal.py:handle_exit():75] Internal process exited
